packages/web3-core/src/web3_subscription_manager.ts

```diff
@@ -126,10 +157,17 @@ export class Web3SubscriptionManager<
 		return Promise.all(result);
 	}
 
+	/**
+	 * Clears all subscriptions
+	 */
 	public clear() {
 		this._subscriptions.clear();
 	}
 
+	/**
+	 *
```

Comment:
```suggestion
	 * Check whether the current provider supports subscriptions.
	 *
```
Could we, please, have a description for all methods even if it is simple? This would give the reader a better experience, I think.